GitUtils.ps1

```diff
@@ -117,45 +117,73 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesUnmerged = @()
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
-            dbg 'Getting status' $sw
-            $status = git -c color.status=false status --short --branch 2>$null
-        } else {

```

Comment:
Yep, but removed intentionally. If I'm reading the original code correctly the empty status set by the else block serves to make the loop no-op (and has no additional references below). In this version the loop will only be hit when settings.EnableFileStatus is true (checked before checking whether to get it from the cache or from "git status), so the else block is now dead code. Please correct me if I'm misunderstanding.